lib/mathjax-helper.coffee

```diff
@@ -43,15 +44,72 @@ module.exports =
     return
 
 #
+# Define some functions to help get a hold of the user's Latex
+# Macros.
+#
+namePattern = ///             # The name of a macro can be either
+              ^[^a-zA-Z\d\s]$ # a single non-alphanumeric character
+              |               # or
+              ^[a-zA-Z]*$     # any number of lower and upper case
+              ///             # letters, but no numerals.
+
+valuePattern = ///^    # The stringified version of the definition
+               {.*}    # must be between brackets
+               (,\d)?  # and may be followed by a comma and a digit
+               $///
+
```

Comment:
These regexs work like I expect them too, but I don't pretend to be a regex expert. If something seems amiss please let me know.